### PR TITLE
Add FAQs

### PR DESCRIPTION
**Pull request by Sbozzolo**
Start collecting common complaints.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,6 +282,18 @@ system.
...
+### My Caltech Box link looks different
+
+Your Box link should look something like
+```
+https://caltech.box.com/shared/static/pdsre5tumpc04qbomzjduw07ryd3emwj.gz
+```
+It has to have a `/shared/static` portion and an extension. If that's not the
+case, chances are you copied the link from the wrong page. Make sure you are in
+the "Link Settings" page (see screenshot above).
+
```
Comment (Julians42) [inline README.md:296]: When this didn't work for me I got a `LoadError` preceded by the following: 
```bash
ERROR: /var/folders/4d/XXX/XX/....
Can not open the file as archive
```
Guessing first time users would not recognize that the error is their box link? I would say the easy check is to `cd` to the artifact directory `~/.julia/artifacts/XXX` and see if the object has actually been downloaded.  Suggest to revise section header as follows (or include in the first sentence or two):
### `ClimaArtifactsHelper` is not able to load my artifact
and then point to checking the caltech box link...

**Review by Julians42 — APPROVED** [review]
This is great! 
One suggestion for the next FAQ:
I ended up copying the template for my download from another folder but it might be nice to point users to an easy first example, i.e., simply saving a file as an artifact. It would look something like this 
```julia
using Downloads 
using ClimaArtifactsHelper

# file url should link directly to the object you want to download. 
const FILE_URL = "https://caltech.box.com/shared/static/cryptic_hash_path.nc"
const FILE_PATH = "ERA5_DATA.nc" # name of the file after downloading, e.g., data from ERA5

output_dir = "era5_data" # descriptive name of the folder where the artifact will be stored
if isdir(output_dir)
    @warn "$output_dir already exists. Content will end up in the artifact and may be overwritten."
    @warn "Abort this calculation, unless you know what you are doing."
else
    mkdir(output_dir)
end

if !isfile(FILE_PATH)
    @info "$FILE_PATH not found, downloading it (might take a while)"
    data_file = Downloads.download(FILE_URL)
    Base.mv(data_file, FILE_PATH)
    Base.cp(FILE_PATH, joinpath(output_dir, basename(FILE_PATH)))
end

@info "Data file generated!"
create_artifact_guided(output_dir; artifact_name = basename(@__DIR__))
```

**Comment by Sbozzolo** [comment]
> This is great! One suggestion for the next FAQ: I ended up copying the template for my download from another folder but it might be nice to point users to an easy first example, i.e., simply saving a file as an artifact. It would look something like this
> 
> ```julia
> using Downloads 
> using ClimaArtifactsHelper
> 
> # file url should link directly to the object you want to download. 
> const FILE_URL = "https://caltech.box.com/shared/static/cryptic_hash_path.nc"
> const FILE_PATH = "ERA5_DATA.nc" # name of the file after downloading, e.g., data from ERA5
> 
> output_dir = "era5_data" # descriptive name of the folder where the artifact will be stored
> if isdir(output_dir)
>     @warn "$output_dir already exists. Content will end up in the artifact and may be overwritten."
>     @warn "Abort this calculation, unless you know what you are doing."
> else
>     mkdir(output_dir)
> end
> 
> if !isfile(FILE_PATH)
>     @info "$FILE_PATH not found, downloading it (might take a while)"
>     data_file = Downloads.download(FILE_URL)
>     Base.mv(data_file, FILE_PATH)
>     Base.cp(FILE_PATH, joinpath(output_dir, basename(FILE_PATH)))
> end
> 
> @info "Data file generated!"
> create_artifact_guided(output_dir; artifact_name = basename(@__DIR__))
> ```

I implemented a simpler version here:
https://github.com/CliMA/ClimaArtifacts/pull/9

Could you try this and see if it works for your artifact? If it does, I'll merge it.

**Comment by Sbozzolo** [comment]
Updated, thank you!

**Pull request merged**